src/rsz/src/Resizer.cc

```diff
@@ -1207,13 +1373,30 @@ void Resizer::resizePreamble()
   checkLibertyForAllCorners();
   findBuffers();
   findTargetLoads();
+  findFastBuffers();
 }
 
 // Filter equivalent cells based on the following liberty attributes:
 // - Footprint (Optional - Honored if enforced by user): Cells with the
 //   same footprint have the same layout boundary.
 // - User Function Class (Optional - Honored if found): Cells with the
 //   same user_function_class are electrically compatible.
+
+bool Resizer::areCellsSwappable(LibertyCell* existing, LibertyCell* replacement)
```

Comment:
@precisionmoon has added area and leakage optional limits to Resizer::getSwappableCells.  Should they apply here too?  If so it would be better to share the logic.

Reply:
Actually I see you are sharing the code but the limits seem to have disappeared in the refactoring